packages/nonMenuDropdown/NonMenuDropdown.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import { useId } from "react-id-generator";
+import FocusLock from "react-focus-lock";
+import { Dropdownable } from "../dropdownable";
+import { Popover } from "../popover";
+import { SpacingBox } from "../styleUtils/modifiers";
+import { DropdownableProps } from "../dropdownable/components/Dropdownable";
+import { ResetButton } from "../button";
+import { display } from "../shared/styles/styleUtils";
+
+interface NonMenuDropdownProps
+  extends Omit<DropdownableProps, "open" | "dropdown" | "onClose"> {
+  /**
+   * A unique identifier to help screenreaders or UI testing tools associate the dropdown content with it's trigger button. If an ID prop is not provided, an arbitrary ID will be generated.
+   */
+  id?: string;
+  /**
+   * Whether the dropdown starts open
+   */
+  initialIsOpen?: boolean;
```

Comment:
how about `isInitiallyOpen`?

Reply:
I was trying to keep it consistent with `DropdownMenu`, which uses `initialIsOpen`